_articles/open-firmware-smmstore.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: article
+title: Firmware Flash to Clear the SMMSTORE ROM (Error code 5)
+description: >
+  How to fix the failed with exit status: exit code: 5 message
+keywords:
+  - Firmware
+  - Open Firmware
+  - SMMSTORE
```

Comment:
UEFI keyword would be useful, as modifying EFI variables are how users will experience this issue.

Reply:
Addressed in commit https://github.com/system76/docs/pull/561/commits/d67b0438bfd7e8dc9c13fb9f8cb6fbd60e47cba1.